arithmetization/src/test/java/net/consensys/linea/zktracer/ReplayTests.java

```diff
@@ -52,6 +53,7 @@
  * One can run this command: scripts/capture.pl --start xxx --end yyy --output my-test-case.json.gz
  */
 @Slf4j
+@Tag("nightly")
```

Comment:
So is the idea that we add this at the top of any class that contains replay tests (once we're done debugging them) ?

Reply:
At the top of the class or specific tests inside classes.